packages/mobile/src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -80,6 +81,7 @@ function NameAndPicture({ navigation }: Props) {
     dispatch(setPromptForno(true)) // Allow forno prompt after Welcome screen
     ValoraAnalytics.track(OnboardingEvents.name_and_picture_set, {
       includesPhoto: false,
```

Comment:
Its a bit confusing leaving this property around. Do we still need it?

Reply:
i think this prop means if whether a profile picture is included. The default `false` is incorrect. I had a fix for it but reverted that b/c it was difficult to unit test pictureInput component. I'm planning on fixing it in a separate PR